experimental/packages/opentelemetry-sdk-metrics-base/examples/README.md

```diff
@@ -0,0 +1,20 @@
+# OpenTelemetry JavaScript Examples
+
+This directory contains examples of how to run real applications with OpenTelemetry JavaScript.
+
+## Work in Progress Examples
+
+These examples are using work in progress metrics packages. 
+
+|Name | Description | Complexity Level |
+------------- | ------------- | ------------ |
+|[metrics](metrics/) | Basic metrics collection and exports those metrics to a Prometheus compatible endpoint | Beginner |
+
+
+## Contributing
+
+Please see [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry-js/blob/main/CONTRIBUTING.md) for instructions on how to contribute.
+
+## LICENSE
+
+Apache License 2.0
```

Comment:
lint: `./experimental/packages/opentelemetry-sdk-metrics-base/examples/README.md:20:18 MD047/single-trailing-newline Files should end with a single newline character`

Reply:
This has been fixed.